ArjunaJTA/jta/pom.xml

```diff
@@ -25,9 +26,9 @@
   <packaging>jar</packaging>
 
   <properties>
-      <server.jvm.args>
-          ${jvm.args.other} ${jvm.args.memory} ${jvm.args.debug} ${jvm.args.modular}
-      </server.jvm.args>
+    <server.jvm.args>
```

Comment:
Part of the reason for asking engineers to minimise reformatting code is that it makes it difficult to track the reasons for fixes. If you really need to reformat the code amongst other functional changes then can you do the reformatting in a separate commit.

Reply:
> I'm not sure we have formally documented that requirement so I will look to add a doc/text explaining the project requirements.

Yes, please. It is not very clear whether we can reformat old code when we carry out functional modifications or if we should apply style rules on a separated commit.